workshops/weather_app/README.md

```diff
@@ -0,0 +1,409 @@
+Personally, I feel cards are very aesthetic form to display our content. Also, there is a lot of scope of making the cards look more better and
+more catchy.
+
+P.S: For more information about cards, one could also use `Bootstrap`, an open source `CSS` framework. 
+[Bootstrap Card](https://getbootstrap.com/docs/4.0/components/card/)
+
+Inside our `<body>` we will add a `<div>` with class as `container` which makes our main card and for the content inside we can add another
+`<div>` with class as `content`.
+Then add a heading using `h1` tag giving the title to the project.
+
+To input the city from the user we use the `input` tag and declare it with class as `input` and `id` as `input`. 
+
+Mainly `class` is used for passing information in `CSS` and `id` is used for `JS`. 
+Also add a `placeholder` which is a text behind the input label that describes the expected value of input.
+
+`<input id="input" class="input" placeholder="Enter the City Name">`
+
+Make sure you close the `div` of `content` because now we have taken the city name from the user and our only task is to display the weather 
+details.
+
+Lastly we have to add the last `div` with class as `main-weather` to display the weather details.
+
+We will be using `<p>` for every detail we display. Hence we declare the details with the following ids
+
+```
+            <p id="date">Date </p>
+            <p id="city">City </p>
+            <p class="temp" id="temp">Temp </p>
+            <p id="min-max">Min and Max Temp </p>
+            <p id="weather-type">Sunny </p>
+```
+
+Our `HTML` code is done and now we can move to the `CSS` part. At the end the `HTML` code will look like:
+
+![Screenshot from 2020-10-09 19-36-08](https://cloud-ad455y4xg.vercel.app/0screenshot_from_2020-10-09_19-36-08.png)
+
+## 4. Improving the design of skeleton using CSS
+
+PRO TIP: Before we start coding for any project, always try to make a flowchart containing all the details. This will make things crystal clear 
+and the chances of any mistakes would also reduce.
+
+Here is a CSS diagram which depicts the details we have to consider.
+
+![CSS(flowchart)](https://cloud-gkcw8pvg7.vercel.app/0css_flowchart_.png)
+
+
+Let's start coding `CSS` by making all the elements with `0` margin and padding. 
+
+Now we will customize the body by giving it a background color and some other details.
+
+Till now the code would look something like this
+
+```
+body {
+  background-color: #dfe7ee;
+  line-height: 1.5;
+  font-size: 125%;
+  display: flex;
+}
+```
+
+All our body part is done, now we can start making the Card.
+
+```
+.container {
+  position: relative;
+  background-color: #fff;
+  padding: 0 1em 1em; 
+  margin: 150px 500px 400px auto; 
+  -webkit-filter: drop-shadow(0 1em 1em rgba(0, 0, 0, 0.1));
+          filter: drop-shadow(0 1em 1em rgba(0, 0, 0, 0.1));
+}
+```
+Making `position:relative` will cause it to be adjusted away from its normal position. Know more about 

```

Comment:
That’s not really how position works here—it’s changing how elements inside it are positioned, namely the psuedoelement. Also, grammar of second sentence is off